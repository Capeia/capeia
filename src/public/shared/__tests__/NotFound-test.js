// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<NotFound>', () => {
  it('renders correctly', () => {
    const Component = require('../NotFound').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
