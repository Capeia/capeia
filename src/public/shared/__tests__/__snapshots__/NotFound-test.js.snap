// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound> renders correctly 1`] = `
<div
  className="container"
>
  <h1>
    404!
  </h1>
  <p>
    Unfortunately, this URL doesn't point anywhere.
  </p>
</div>
`;
